### ISSUE TITLE
Stadia SDK

### ISSUE THREAD

**Issue by laghatelyan** [googlestadia/vsi-lldb#4]
Hello,

I wanted to use this library for remote debugging, but I encountered an issue. It seems that Stadia has ceased operations, and it is no longer possible to obtain the SDK. Is there anyone who could share the SDK with me or advise on how and where to download it now?

**Comment by werat** [comment]
Hi, unfortunately it's not possible anymore to obtain Stadia SDK. This repository is not maintained and hasn't been updated for a while as well, so it could be considered for "informational purposes".

Stadia SDK is not a "critical" dependency for the Visual Studio extension, however, it is used specifically handling Stadia-specific aspects of setting up the remote debugging (checking permissions, selecting the remove machine, setting up SSH tunnel). It would be possible to remove this dependency and use regular SSH, without affecting the rest of the project.

If you're intrerested in doing so, I could give a few pointers where to start, however, I understand this might be way out of scope of what you want to do :)

**Comment by laghatelyan** [comment]
Yes, I understand and did some research on using the Stadia SDK in the extension, but didn't go into detail thinking it would be easier to find the SDK.

However, since you are answering, I wanted to ask:
I found an article on your blog (https://werat.dev/blog/debugging-wine-with-lldb-and-vscode) in which you describe how to debug wine applications using lldb and a plugin for VS Code. I tried to assemble everything and use VS Code and it seemed to me that it would be more convenient for me to work in MS Visual Studio, so I decided to try to assemble this extension.

Do I understand correctly that this extension will also add the ability to debug an application under Wine in MS VS or am I wrong?

**Comment by werat** [comment]
> Do I understand correctly that this extension will also add the ability to debug an application under Wine in MS VS or am I wrong?

Yes, `vsi-lldb` is an extension for Visual Studio that implements a debug engine that uses LLDB as a debugger. It's original use case was to perform remote debugging of linux processes. So if you manage to build & run the extension, then you would be able to debug in the following setup:
* Visual Studio & LLDB are running on a Windows machine
* Wine application is running on a Linux machine
* Visual Studio debugs the remote process over SSH connection

That said, if you goal is to debug Wine, I would recommend going with VSCode as described in https://werat.dev/blog/debugging-wine-with-lldb-and-vscode. That's much-much easier to set up.

**Comment by AvishakeMitra** [comment]
@werat can you tell me how Build yours's LLDB, the step here is not working for me.

**Comment by werat** [comment]
> [@werat](https://github.com/werat) can you tell me how Build yours's LLDB, the step here is not working for me.

I’ve described the steps how I built it here — https://werat.dev/blog/debugging-wine-with-lldb-and-vscode, see “Building LLDB” section.

The LLVM code base has evolved quite a bit since then, so if you’re trying to build with newer version, the patches may need to be updated.

**Comment by AvishakeMitra** [comment]
@werat , Thanks for ur reply. If we are using **git clone release/14.x --single-branch**, then it should ideally work without any issue ?

**Comment by werat** [comment]
Yes, that should work without issues
